infra/cifuzz/cifuzz.py

```diff
@@ -110,19 +119,21 @@ def build_fuzzers(project_name, project_repo_name, commit_sha, git_workspace,
   return True
 
 
-def run_fuzzers(project_name, fuzz_seconds, out_dir):
+def run_fuzzers(project_name, fuzz_seconds, workspace):
   """Runs all fuzzers for a specific OSS-Fuzz project.
 
   Args:
     project_name: The name of the OSS-Fuzz project being built.
     fuzz_seconds: The total time allotted for fuzzing.
-    out_dir: The location in the shared volume to store output artifacts.
+    workspace: The location in a shared volume to store a git repo and build
+      artifacts.
 
   Returns:
     (True if run was successful, True if bug was found).
   """
-  if not out_dir or not os.path.exists(out_dir):
-    logging.error('Unreachable out_dir argument %s.', format(out_dir))
+  out_dir = os.path.join(workspace, 'out')
+  if not os.path.exists(out_dir):
```

Comment:
Is the only place this is called from `main`?
I might be wrong here, but IMO the program seems a little too defensive in programming style.
We create the directory in `main` so the check here is unnecessary.
You can either remove the creation there or the check here.

Reply:
It is only called from main, but I want it to be self contained.